packages/libraries/core/package.json

```diff
@@ -41,6 +41,7 @@
     "@fluentui-react-native/persona-coin": "0.12.29",
     "@fluentui-react-native/pressable": "0.9.65",
     "@fluentui-react-native/radio-group": "0.16.42",
+    "@fluentui-react-native/switch": "0.8.36",
```

Comment:
nit: we may want to leave this out since I think it will bloat people's packages downstream.